shared/src/business/useCases/pendingItems/generatePrintablePendingReportInteractor.js

```diff
@@ -29,11 +29,21 @@ exports.generatePrintablePendingReportInteractor = async ({
     judge = decodeURIComponent(judge);
   }
 
-  const pendingItems = await applicationContext
-    .getUseCaseHelpers()
-    .fetchPendingItems({ applicationContext, docketNumber, judge });
+  let pendingDocuments = [];
 
-  const formattedPendingItems = pendingItems.map(pendingItem => ({
+  if (docketNumber) {
+    pendingDocuments = await applicationContext
+      .getUseCaseHelpers()
+      .fetchPendingItemsByDocketNumber({ applicationContext, docketNumber });
+  } else {
+    pendingDocuments = (
+      await applicationContext
+        .getUseCaseHelpers()
+        .fetchPendingItems({ applicationContext, judge })
```

Comment:
I'm ignorant about interactors, but would you be able to use the fetchPendingItemsInteractor here?